packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-feedback-modal.pcss

```diff
@@ -1,29 +1,18 @@
 @import '../../global/global.pcss';
 
-[part='reason-title'],
```

Comment:
I removed the mobile look for the feedback modal (it had expanded buttons and titles) and I increased the gap between the buttons on mobile.